x/docs/client/cli/query.go

```diff
@@ -117,3 +121,38 @@ func GetCmdSharedDocuments(cdc *codec.Codec) *cobra.Command {
 		},
 	}
 }
+
+// ----------------------------------
+// --- DocumentReceipt
+// ----------------------------------
+
+func GetCmdReceivedReceipts(cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "receipts [user-address] [doc-uuid]",
+		Short: "Get the document receipt associated with given document uuid",
+		Args:  cobra.RangeArgs(1, 2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cliCtx := context.NewCLIContext().WithCodec(cdc)
+
+			var addr, uuid string
+
+			if len(args) == 1 {
+				addr = args[0]
+				uuid = ""
+			} else {
+				addr = args[0]
+				uuid = args[1]
+			}
```

Comment:
I think we can simplify the aboe `if else` by taking into consideration two things: 

1. The default value for `string`s is `""` in Go. 
2. The first parameter (`args[0]`) will always be present

That being said, we can simplify lines 137-145 to be: 

```golang
addr, uuid := args[0], ""
if len(args) == 2 {
  uuid = args[1]
}
```